### PR TITLE
Chapter 8

### PR DESCRIPTION
**Pull request by phamducpho1**

### CODE REVIEW

**Comment by phamducpho1** [comment]
image

**Comment by phamducpho1** [comment]
ready

File: app/helpers/sessions_helper.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by(id: session[:user_id])
+  end
+
+  def logged_in?
+    !current_user.nil?
```
Comment (dothidiemthao) [inline app/helpers/sessions_helper.rb]: dùng "current_user.present?" thay cho "!current_user.nil?"

**Comment by dothidiemthao** [comment]
commented

File: app/helpers/sessions_helper.rb (hunk 1 above)
Comment (phamducpho1) [reply app/helpers/sessions_helper.rb]: dạ ok chị

**Comment by phamducpho1** [comment]
image

**Comment by phamducpho1** [comment]
ready

**Review by dothidiemthao — APPROVED** [review]

**Comment by dothidiemthao** [comment]
merged

**Pull request merged**